packages/mobile/src/components/now-playing-drawer/TrackInfo.tsx

```diff
@@ -42,12 +42,12 @@ export const TrackInfo = ({
     <View style={styles.root}>
       {user && track ? (
         <>
-          <Pressable onPress={onPressTitle}>
+          <TouchableOpacity onPress={onPressTitle}>
```

Comment:
what's the idea here? just better looking ui?

Reply:
Yeah it felt weird to have the pressable things with no feedback